public/index.html

```
@@ -1797,6 +1840,46 @@ <h1>PTZ Preset Control</h1>
+          <div style="position:absolute; top:50%; left:2px; right:2px; height:1px; background:rgba(255,255,255,0.18); transform:translateY(-50%);"></div>
+          <div id="fullscreen-zoom-thumb" style="position:absolute; top:50%; left:50%; width:44px; height:44px; transform:translate(-50%, -50%); border:1px solid var(--border); background:var(--surf); color:var(--accent); border-radius:14px; box-shadow:0 2px 8px rgba(0,0,0,0.28); cursor:grab;"></div>
+        </div>
+        <div style="position:absolute; font-size:20px; font-weight:700; color:var(--accent); bottom:-20px;">−</div>
+      </div>
+      <div style="font-size:0.75rem; color:var(--muted); font-weight:600;">Zoom</div>
+    </div>
+  </div>
+  <!-- Camera/Cut buttons at bottom -->
+  <div style="padding:12px 16px; border-top:1px solid var(--border); background:var(--surf); flex-shrink:0; display:flex; gap:8px; flex-wrap:wrap; justify-content:center;">
+    <div style="display:flex; gap:6px; flex-wrap:wrap; justify-content:center;" id="fullscreen-camera-buttons"></div>
+    <div style="display:flex; gap:6px;" id="fullscreen-cut-buttons" style="display:none;"></div>
```

HTMLHint (1.9.2)

[error] Duplicate of attribute name [ style ] was found.

(attr-no-duplication)